src/nvim/ex_docmd.c

```diff
@@ -5077,7 +5077,10 @@ static int get_tabpage_arg(exarg_T *eap)
       tab_number = 0;
     } else {
       tab_number = (int)eap->line2;
-      if (!unaccept_arg0 && *skipwhite((char_u *)(*eap->cmdlinep)) == '-') {
+      char *cmdp = strstr(*eap->cmdlinep, eap->cmd);
+      assert(cmdp != NULL);
+      while (--cmdp > *eap->cmdlinep && *cmdp == ' ');
```

Comment:
```suggestion
      while (--cmdp > *eap->cmdlinep && *cmdp == ' ') {}
```

Reply:
Thanks. I thought I could just commit your suggestion and then squash merge in the end, but apparently the commit linter checks every single commit, so.. I'll just force-push again.

Reply:
The commit linter is optional, though, so you can just ignore it until you squash merge.